tests/issues.targets

```diff
@@ -74,12 +74,6 @@
         <ExcludeList Include="$(XunitTestBinBase)/tracing/tracecontrol/tracecontrol/*">
             <Issue>20299</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Interop/NativeLibraryResolveEvent/ResolveEventTests/*">
-            <Issue>23941</Issue>
-        </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Interop/NativeLibraryResolveCallback/CallbackTests/*">
```

Comment:
Are these tests affected by the change below, or should they have been reenabled earlier?  Did you mean to reenable the Callback*Stress*Test line below?

Reply:
These should have been enabled earlier (bug is fixed).

> Did you mean to reenable the CallbackStressTest line below

Thanks for pointing this out. I didn't notice this one was also disabled.